apps/web/src/components/HeroText.module.css

```
@@ -0,0 +1,202 @@
+ */
+.figure {
+  display: inline-flex;
+  align-items: flex-end;
+  font-family: var(--font-sans);
+  font-weight: 800;
+  font-size: clamp(2.75rem, 12vw, 5.5rem);
+  line-height: 1;
+  letter-spacing: -0.04em;
+  color: var(--sea-ink-soft);
+
+  opacity: 0;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)